app/views/rubygems/_rubygem.html.erb

```diff
@@ -2,7 +2,12 @@
   <span class="gems__gem__info">
     <h2 class="gems__gem__name">
       <%= rubygem.name %>
-      <span class="gems__gem__version"><%= latest_version_number(rubygem) %></span>
+      <span class="gems__gem__version">
+        <%= latest_version_number(rubygem) %>
+      </span>
+      <span class="gems__gem__latest__release__date">
+        <%= "released #{latest_release_date(rubygem)}" %>
```

Comment:
`released` should be moved into config/locales/en.yml. Would it be possible to reuse `version_date_tag` from `VersionsHelper`?

Reply:
Not 100% sure what to do with config/locales/en.yml to get the "released" text displayed.
Took a look at https://github.com/rubygems/rubygems.org/blob/0c48ef3d8584e6cc3c8f28e9124f8e7c2be913b2/app/helpers/versions_helper.rb#L18